### ISSUE TITLE
mastodon does not follow prefers color scheme

### ISSUE THREAD

**Issue by 2-www** [mastodon/mastodon#19231]
### Steps to reproduce the problem

1. visit a mastodon link where you aren't logged in while using light theme

### Expected behaviour

dark text on light background

### Actual behaviour

mastodon's built in light theme is used, which is the case when you are visiting a link to a website where you are logged in

### Specifications

mastodon.social, webkit on debian

**Comment by www-cat** [comment]
related: #10976, but as i understand it is about the /web/ interface

**Comment by rachellawson** [comment]
Genuinely surprised this wasn't already a done deal, tbh. The usual options are "light", "dark", "follow system colours" so shall we go with that?

**Comment by bondolo** [comment]
Probably worth supporting `[prefers-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast)` at the same time.

**Comment by cassidyjames** [comment]
Yeah I think this is ultimately a duplicate of #10976.

**Comment by vmstan** [comment]
Should be closed by #29748

**Comment by renchap** [comment]
Duplicate of #10976
